webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -48,35 +51,56 @@ def get_spotify_artist(artist_name):
     Authenticate then search an artist based on an artist name. Searches for the name if id is not provided
     '''
 
-    if len(artist_name) != 22:
-        artist_id = search_spotify_artist(artist_name)
+    #TODO 
+    #Spotify artist is a 22 fixed character ID, but searching for a word would require to first call search_spotify_artist 
+    #to get the id which would also return some ids even if not an artist. Not sure how to handle this.
 
-    headers = spotify_authenticate()
+    if len(artist_name) != SPOTIFY_FIXED_ARTIST_CHAR_LEN:
+        artist_id = search_spotify_artist(artist_name)
```

Comment:
I think we need to move this out of this function to first have the user choose the row / id, then call `get_spotify_artist` for that specific one

Reply:
Probably would first put the text field and then get the dropdown based on the options.